.github/workflows/build-sw-container.yml

```diff
@@ -16,53 +16,46 @@ jobs:
     - name: make date tag
       id: mkdatetag
       run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
-
+    
```

Comment:
Some extra whitespace snuck in here, can you find the offending commit and rebase it out?